stopwatch-JSF/prompts.md

```
@@ -0,0 +1,262 @@
+# Core functionalities:
+* There should be a menu page with two options "Stopwatch" and "Countdown".
+* When clicking on "Stopwatch", a new page should be opened with the stopwatch functionality which is already implemented in the index.html, scripts.js and styles.css files that are attached.
+* When clicking on "Countdown", a new page should be opened with the countdown functionality, which should be implemented from scratch taking into consideration the following requirements:
+   * Display time in HH:MM:SS starting with 00:00:00.
+   * Decide the best user experience for the user to enter the time for the countdown. For example, it could be set of buttons from 0 to 9.
+   * There should be there buttons: "Start", "Pause" and "Clear"
+      * The "Start" button starts the countdown.
+      * The "Pause" button pauses the countdown. When clicked, it should change its label to "Resume", allowing the user to continue the countdown from where it was paused. When "Resume" is clicked, it's label should change back to "Pause".
+      * The "Clear" button stops the countdown and sets the time back to 00:00:00.
+   * Display an alert when the countdown finishes. The alert's message is "Done! Do whatever you need to do".
+#Code organization:
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)